fake.js

```diff
@@ -59,6 +59,9 @@ module.exports = class FakeTransaction extends Transaction {
    * @return {Buffer}
    */
   hash (includeSignature) {
+    if (includeSignature === null || includeSignature === undefined) {
+      includeSignature = true
+    }
```

Comment:
Why you replace `null` to `true`?
Right check will as in `Transaction#hash`: https://github.com/ethereumjs/ethereumjs-tx/blob/6ea5b5030fb6a7f9786ef7fb158c0593a517fef7/index.js#L144

Since package expects ES6, we also can use default function parameters:
```js
  hash (includeSignatures = true) {
    ...
  }
```

Reply:
@fanatid Thanks for the feedback. I just saw that the package doesn't have any compilation/build steps and assumed it's gonna be better to stick to ES5 in that case, but if we're OK with that - changed.